modules/currentTime.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-return-assign */
+import { DateTime } from './luxon.js';
+
+export const displayTime = document.querySelector('.current-time');
+
+export function time() {
+  const dt = DateTime.local();
+  return displayTime.textContent = dt.toFormat('MMMM d, yyyy - hh:mm:ss a');
+}
```

Comment:
- [ ] Please change this function to an arrow function to follow the ES6 syntax as requested in the project requirements.
  - You can consider using the below code.
```ruby
export const time = () => {
  const dt = DateTime.local();
  return displayTime.textContent = dt.toFormat('MMMM d, yyyy - hh:mm:ss a');
}